effects/netlify/default.nix

```
@@ -18,7 +19,7 @@ let
 in
 mkEffect (args // {
   inputs = [ netlify-cli ];
-  secretsMap."netlify" = secretName;
+  secretsMap = secretsMap // { "netlify" = secretName; };
```

```suggestion
  secretsMap = { "netlify" = secretName; } // secretsMap;
```

Allows only `secretsMap` to be specified, making it more like `mkEffect`.